### PR TITLE
Update feature_request.md

### PR DESCRIPTION
**Pull request by bgavrilMS**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -4,14 +4,9 @@ about: Suggest an idea for this project
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+**MSAL is the preferred auth library for Microsoft**
```
Comment (jennyf19) [inline .github/ISSUE_TEMPLATE/feature_request.md]: idk about this statement..maybe something like "MSAL is the recommended/preferred auth library for use with the Microsoft identity platform"...not sure.

**Review by jennyf19 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (henrik-me) [reply .github/ISSUE_TEMPLATE/feature_request.md]: I like the suggestion

---
In reply to: [299109879](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1623#discussion_r299109879) [](ancestors = 299109879)

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -4,14 +4,9 @@ about: Suggest an idea for this project
...
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+No new features will be implemented on ADAL. The team's efforts are on improving (MSAL)[https://github.com/AzureAD/microsoft-authentication-library-for-dotnet], the next-gen auth library. MSAL's wiki contains a (migration guide)[https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Adal-to-Msal] from ADAL.
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+If you require a new feature, there's a good chance it has already been implemented in MSAL. If it hasn't, tell us about it and we will prioritize it!
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
```
Comment (henrik-me) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]: I suggest pointing to the bug template... 

Wanted to open a bug? Please open an issue [here](link) to let us know the details.

**Review by henrik-me — APPROVED** [review]
:shipit:

**Pull request merged**